lib/howl/commands/edit_commands.moon

```diff
@@ -98,6 +98,25 @@ command.register
       app.editor.line_at_top = replacement.line_at_top
 
 command.register
+  name: 'buffer-sort-lines'
+  description: 'Sort the selected lines or the entire buffer'
+  handler: ->
+    editor = howl.app.editor
+    local lines
```

Comment:
You wouldn't need the `local lines` in that case. FWIW, I think it's fine as is as well, even though I've used the if-statement-as-expression myself it reads a little bit funny to me.